src/remake.jl

```diff
@@ -69,10 +69,10 @@ function remake(prob::ODEProblem; f = missing,
             u0 = prob.u0
         end
         if (eltype(p) <: Pair && !isempty(p)) || (eltype(u0) <: Pair && !isempty(u0)) # one is a non-empty symbolic map
-            hasproperty(prob.f, :sys) && hasfield(typeof(prob.f.sys), :ps) ||
+            isempty(parameter_symbols(prob)) &&
```

Comment:
can we not use parameter_symbols here? it can be relatively expensive.

Reply:
No, there almost always is a `SymbolCache` but it may or may not have symbols of a particular type

Reply:
Fixed